project_report/tex/implementation.tex

```diff
@@ -86,3 +86,170 @@ \subsection{OPC-UA-Client}
 subscriptions will add data to the object. When done the Batch object will run
 through its logic and finally export itself as a JSON so it can be sent to the
 API. 
+
+
+\subsection{Overall Equipment Effectiveness}
+When a batch is complete, the calculation of the OEE takes place, and the data
+stored in the batch class is used for this.\\
+
+For this project, the planned production time can not be estimated properly, as
+too many variables are unknown (shift lengths, the time it takes to refill
+ingredients, the time it takes to do maintenance work etc.). Therefore, it has
+been chosen to use the time it takes to produce a batch. The calculation can be
```

Comment:
chosen = decided

maybe: "...decided to ONLY use the time it takes to produce a batch."